lib/clearance/configuration.rb

```diff
@@ -93,6 +98,7 @@ def initialize
       @cookie_name = "remember_token"
       @httponly = false
       @mailer_sender = 'reply@example.com'
+      @password_reset_time_limit = 15.minutes
```

Comment:
Clearance users are free to configure this to use an environment variable in their initializer, but we have to provide a valid default from the gem.